app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Comments should belong to an author, not user. Kindly fix that. You can use the `class_name` property to point Rails to the User model.

  ```
  belongs_to :author, class_name: 'User'
  ```